andela/apollo-ah-frontend
# #165633288 signup autologin

**What does this PR do?**
This PR enables a user to get automatically logged in when he signup

**Tasks to be completed?**
- Fix data gotten from the backend (token)
- decode token
- store  user data in the store

**How can this be manually tested?**
- clone the branch
- install node and  yarn 
- run yarn install
- run yarn run start:dev
- navigate to the signup page and signup, you should be automatically logged in.

**Relevant PT stories?**
`165633288`
